distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/JobManagerPaths.java

```diff
@@ -3,88 +3,100 @@
 import org.apache.curator.utils.ZKPaths;
 
 class JobManagerPaths {
-    public static final String WORK_POOLED_JOB_ID = "work-pooled";
-    public static final String WORK_POOL = "work-pool";
-    public static final String REGISTRATION_VERSION = "registration-version";
-    public static final String ASSIGNMENT_VERSION = "assignment-version";
+    private static final String ALIVE = "alive";
+    private static final String ASSIGNMENT_VERSION = "assignment-version";
+    private static final String LEADER_LATCH = "leader-latch";
+    private static final String LOCKS = "locks";
+    private static final String REGISTRATION_VERSION = "registration-version";
+    private static final String WORKERS = "workers";
+    private static final String ASSIGNED = "assigned";
+    private static final String AVAILABLE = "available";
+    private static final String WORK_POOLED_JOB_ID = "work-pooled";
+    private static final String WORK_POOL = "work-pool";
 
     final String rootPath;
 
     JobManagerPaths(String rootPath) {
         this.rootPath = rootPath;
     }
 
-    String getAssignmentVersion() {
-        return ZKPaths.makePath(rootPath, ASSIGNMENT_VERSION);
-    }
-
-    String getRegistrationVersion() {
-        return ZKPaths.makePath(rootPath, REGISTRATION_VERSION);
-    }
-
-    String getWorkersPath() {
-        return ZKPaths.makePath(rootPath, "workers");
+    public String getWorkersAlivePath() {
+        return path(ALIVE);
     }
 
     String getWorkerAliveFlagPath(String workerId) {
-        return ZKPaths.makePath(getWorkersAlivePath(), workerId);
+        return path(ALIVE, workerId);
     }
 
-    String getWorkerPath(String workerId) {
-        return ZKPaths.makePath(getWorkersPath(), workerId);
+    String getAssignmentVersion() {
+        return path(ASSIGNMENT_VERSION);
     }
 
-    String getWorkerAvailableJobsPath(String worker) {
-        return ZKPaths.makePath(getWorkerPath(worker), "available");
+    String getLeaderLatchPath() {
+        return path(LEADER_LATCH);
     }
 
-    String getWorkerAssignedJobsPath(String worker) {
-        return ZKPaths.makePath(getWorkerPath(worker), "assigned");
+    String getWorkPooledLocksPath() {
+        return path(LOCKS, WORK_POOLED_JOB_ID);
```

Comment:
Since we are changing djm paths: let's move locks from   
/locks/work-pooled/my-job-id/my-work-item-1.lock     
to  
/locks/my-job-id/my-work-item-1.lock